cd/cd_dump.ixx

```diff
@@ -423,7 +423,7 @@ export bool redumper_dump_cd(Context &ctx, const Options &options, bool refine)
 
         bool read_cdtext = !options.disable_cdtext;
         // disable multisession CD-TEXT for certain drives that hang indefinitely
-        if(toc.sessions.size() > 1 && ctx.drive_config.vendor_id == "PLEXTOR" && ctx.drive_config.product_id == "CD-R PX-W4824A")
+        if(toc.sessions.size() > 1 && ctx.drive_config.vendor_id == "PLEXTOR" && ctx.drive_config.product_id == "CD-R PX-W4824A" && !options.enable_multisession_cdtext)
```

Comment:
Please change it to:
if((toc.sessions.size() > 1 || options.enable_multisession_cdtext) && ctx.drive_config.vendor_id == "PLEXTOR" && ctx.drive_config.product_id == "CD-R PX-W4824A")

Makes the check more obvious.